swagger.yml

```diff
@@ -315,15 +315,15 @@ paths:
         List APIs, in OAS fromat
          Only if used without the Tyk Dashboard
       tags:
-        - APIs
+        - OASAPIs
       operationId: listApisOAS
       parameters:
-      - description: mode of export
-        name: mode
-        in: query
-        schema:
-          type: string
-        example: public
+        - description: mode of export
```

Comment:
can we extend a bit on this, explaining what are the mdodes?

Reply:
done